src/preferenceCenter/preferenceCenter.blocks.js

```diff
@@ -0,0 +1,199 @@
+      },
+      content:
+        `<div data-slot="channelfrequency">
+          <table class="table table-striped" draggable="false">
+            <tbody>
+              <tr draggable="false">
+                <td draggable="false">
+                  <div class="text-left" draggable="false">
+                    <input type="checkbox" checked="" draggable="false">
+                    <label class="control-label" draggable="false">
+                        ${Mautic.translate('grapesjsbuilder.preferenceCenter.block.channelFrequency.text.contact_through')}
+                    </label>
+                  </div>
+                </td>
+              </tr>
+              <tr draggable="false">
+                <td draggable="false">
+                  <div id="frequency_email" class="text-left row" draggable="false">
+                    <div draggable="false">
+                      <label draggable="false">${Mautic.translate('grapesjsbuilder.preferenceCenter.block.channelFrequency.text.do_not_send')}</label>
+                      <input type="text" class="frequency form-control" draggable="false">
+                      <label class="fw-n frequency-label" draggable="false">${Mautic.translate('grapesjsbuilder.preferenceCenter.block.channelFrequency.text.each')}</label>
+                      <select class="form-control" draggable="false">
+                        <option value="" selected="selected" draggable="false"></option>
+                      </select>
+                    </div>
+                    <div draggable="false">
+                      <label draggable="false">${Mautic.translate('grapesjsbuilder.preferenceCenter.block.channelFrequency.text.pause_from')}</label>
+                      <input type="date" class="form-control" draggable="false">
+                      <label class="frequency-label fw-n" draggable="false">${Mautic.translate('grapesjsbuilder.preferenceCenter.block.channelFrequency.text.to')}</label>
+                      <input type="date" class="form-control" draggable="false">
+                    </div>
+                  </div>
+                </td>
+              </tr>
+            </tbody>
+          </table>
+        </div>`,
+      style: { padding: '50px' },
+    });
+
+    this.blockManager.add('UnsubscribeAll', {
+      tagName: 'div',
+      label: Mautic.translate('grapesjsbuilder.preferenceCenter.block.unsubscribeAll.label'),
+      category: Mautic.translate('grapesjsbuilder.preferenceCenterLabel'),
+      draggable: false,
+      attributes: {
+        class: 'fa fa-ban'
+      },
+      content:
+        `<div data-slot="donotcontact">
+            <a href="{dnc_url}" draggable="false">${Mautic.translate('grapesjsbuilder.preferenceCenter.block.unsubscribeAll.link')}</a> 
+            <span draggable="false"> ${Mautic.translate('grapesjsbuilder.preferenceCenter.block.unsubscribeAll.text')}</span>
+        </div>`,
+      style: { padding: '50px' },
+    });
+
+    const style = `<style>
```

Comment:
According to grapesjs doc we should avoid putting styles in blocks. https://grapesjs.com/docs/modules/Blocks.html#important-caveats

would it work if we add it to the  (https://github.com/mautic/mautic/blob/5.x/plugins/GrapesJsBuilderBundle/Assets/library/js/grapesjs-custom.css)